replibyte/src/tasks/full_dump.rs

```diff
@@ -18,17 +18,19 @@ where
     source: S,
     datastore: Box<dyn Datastore>,
     options: SourceOptions<'a>,
+    arg_buffer_size: usize,
 }
 
 impl<'a, S> FullDumpTask<'a, S>
 where
     S: Source,
 {
-    pub fn new(source: S, datastore: Box<dyn Datastore>, options: SourceOptions<'a>) -> Self {
+    pub fn new(source: S, datastore: Box<dyn Datastore>, options: SourceOptions<'a>, arg_buffer_size: usize) -> Self {
```

Comment:
My 2 comments:
1. Include `arg_buffer_size` into `options` and remove `arg_buffer_size` parameter
2. rename `options.arg_buffer_size` into something more explicit like `dump_chunk_size` or anything else